config/values.yaml

```diff
@@ -28,7 +28,10 @@ sync:
       rewriteHosts:
         enabled: true
         initContainer:
-          image: "library/alpine:3.20"
+          image:
+            registry: ""
+            repository: library/alpine
+            tag: "3.20"
```

Comment:
Do you see a way to reuse the vCluster image here? Then we wouldn't even need that option and a separate image anymore?

Reply:
I can imagine users don't want a >300M image where <10M suffice.
If we mirrored alpine or busybox, we could replace the docker hub reference.